src/Assets/age.tsx

```diff
@@ -11,18 +12,24 @@ export function getCurrentMonthYear() {
   };
 }
 
-export function hasBirthMonthYear(birthYear: number, birthMonth: number) {
+type MemberBirthMonthYear = Required<Pick<HouseholdData, "birthMonth" | "birthYear">>
+
+export function hasBirthMonthYear(member: MemberBirthMonthYear | Partial<HouseholdData>): member is MemberBirthMonthYear {
+  const {birthMonth, birthYear} = member;
   const hasBirthYear = birthYear !== undefined && birthYear !== null;
   const hasBirthMonth = birthMonth !== undefined && birthMonth !== null;
   return hasBirthYear && hasBirthMonth;
 }
 
-export function calcAge(age: number, birthYear: number, birthMonth: number) {
-  const hasBirthDate = hasBirthMonthYear(birthYear, birthMonth);
+export function calcAge(member: Partial<HouseholdData>) {
```

Comment:
Can you just use `HouseholdData` here?

Reply:
@CalebPena Thanks for the suggestion, I cleaned up the types.